CMakeLists.txt

```diff
@@ -239,7 +239,22 @@ if (USE_BOOST)
   find_package (Boost 1.46 ${BOOST_REQUIRED_ARG})
   if (Boost_FOUND)
     set (OPENTURNS_HAVE_BOOST TRUE)
-
+    if (USE_MPFR)
+      find_package (MPFR)
+      if (MPFR_FOUND)
+        set (OPENTURNS_HAVE_MPFR TRUE)
+        list (APPEND OPENTURNS_PRIVATE_INCLUDE_DIRS ${MPFR_INCLUDE_DIRS})
+        list (APPEND OPENTURNS_PRIVATE_LIBRARIES ${MPFR_LIBRARIES})
+      endif ()
+    endif ()
+    if (USE_MPC)
```

Comment:
The USE_MPFR/USE_MPC are missing on top of that file, so currently it wont be enabled by default

Reply:
also the dependencies list must be updated in the dev doc